### ISSUE TITLE
NDI Audio missing on recording, visible in OBS Mixer. (incl. workaround)

### ISSUE THREAD

**Issue by Trouffman** [Palakis/obs-ndi#592]
Hi folks,

Not posting much publicly, but this might be useful for more people leveraging NDI.

**Issue** : 
- NDI Source in OBS shows audio in VU Meter
- Audio is not in the recording

**Context** :
- This behavior is when you first boot OBS.
- This is with the NDI-Source Bandwidth settings as : Audio-Only

**How to reproduce ?**
- Open OBS
- Add new NDI source with bandwidth settings : audio-only
- close OBS
- open OBS
- start recording
- stop recording
- listen to the recording (and there is no sound)

Video walkthrough of the bug + how to reproduce the bug + workaround : https://vimeo.com/483118956/bb5fd21564

**Other possibly related bugs :**
Audio shows up in the monitor levels, but the scene is muted when recorded. : #429 
_Most likely the same bug but tested locally (vs NDI-HX)

NDI HX Audio glitches : #448 
NDI OBS! NDI record sound problem : #565 

**Test setup :** 
- All tests to report thi bug where done locally (to remoe network-based issues)
- Windows 10 (all up-to-date)
- OBS 26.0.2
- NDI-OBS (using ndi SDK 4.5.1.0)

**Comment by Trouffman** [comment]
Another similar issue : 
https://github.com/Palakis/obs-ndi/issues/621

**Comment by paulpv** [comment]
Palakis installs v4.5.1 of the 
https://github.com/Palakis/obs-ndi/blob/master/installer/installer.iss
```
#define NDIRuntimeVersion "4.5.1"
  ndiRedistURL = 'https://ndi.palakis.fr/runtime/ndi-runtime-{#NDIRuntimeVersion}-Windows.exe';
```
The latest as of this moment is 4.6.1, which he has on his share:
https://ndi.palakis.fr/runtime/

You can try installing that and report back here.

I installed the full latest NewTek NDI Tools and SDK and rebooted and my problem with this went away.

**Comment by coldfire99** [comment]
I have the same issue but installing 4.6.1 Runtime does not change anything. There is still no audio in the recordings.

**Comment by Trouffman** [comment]
Update on this bug.

So if OBS is launched with an NDI source in "Audio Only", this will use the "audio only option" for geting the NDI Stream, That source's audio leve will show up in the monitor but NOT on the track monitor (when using OBS Music Edition.

This is happenning after Audio decoding but before being added to the audio track.

**Comment by Trouffman** [comment]
This is now fixed in the #849 PR.

To use audio-only you MUST use NDI 5.5 runtime + obs-ndi 4.11 or above + set "sync" to "network".
![image](https://user-images.githubusercontent.com/2151317/222437027-35370620-a1a1-4deb-bda3-778906f5d4b6.png)

**Comment by Trouffman** [comment]
Fix found !
https://github.com/obs-ndi/obs-ndi/issues/429#issuecomment-2115155392

PR Coming soon
